docs/semgrep-app/findings.md

```diff
@@ -12,25 +12,44 @@ import MoreHelp from "/src/components/MoreHelp"
+A **finding** is the core result of Semgrep's analysis. Findings are generated when a Semgrep rule matches a piece of code.
 
-Types of findings:
+A finding can be categorized **based on the issue or code it detects**:
 
 * anti-patterns
-* security vulnerabilities, business, or logic bugs
-* audits for dangerous function use
-* reviewing authentication or authorization logic
-* custom pattern matches based on your own custom rules
+* security vulnerabilities (such as dangerous function usage)
+* business or logic bugs
+* matches based on your own custom rules (such as organization-specific authentication logic)
 
+Semgrep rules provide a metadata schema to identify common categories such as the above. Semgrep findings include a `message` field that describes the security issue or bug that must be resolved. Findings may also provide an `autofix` field that fixes the issue by creating a suggestion within your source code management (SCM) tool, such as GitHub or GitLab.
 
-After a finding is generated, developers may:
+Another way of categorizing findings is **based on the validity of of the match**:
 
-* View the Semgrep rule and the matching code.
-* Fix the issue detected by the finding.
-* Ignore the finding.
-* Comment on the finding.
-* Create a Jira ticket from the finding (for Enterprise/Team Tier users.)
+<dl>
+    <dt>True positive</dt>
+    <dd>Rules are written to match a certain code pattern. A true positive is a genuine match. The rule is capturing code as intended.</dd>
+    <dt>False positive</dt>
+    <dd>A false positive is a mismatch between the intended purpose of the rule and the code it matched. A finding is generated but does not meet the rule's intended need. Rules with a high false positivity rate are said to be <strong>noisy</strong>.</dd>
+    <dt>False negative</dt>
+    <dd>A false negative is a finding that should have been found by a rule, but was not. This could be due to a flaw in the rule's logic. See <a href="../release-notes/#additions-1"><code>semgrep shouldafound</code></a>.</dd>
+</dl>
 
+## Working with findings
+
+After a finding is generated, developers can:
+
+* **Fix the issue detected by the finding.** This is Semgrep's primary goal. In this case, the rule produces a **true positive** finding (such as a security issue) as intended and developers must refactor or address the code such that the rule no longer matches it.
+* **View the Semgrep rule and the matching code.** For developers aiming to understand their team's security posture, Semgrep provides a top-level report view through the Dashboard and a list view of findings in the Findings page that can be filtered by repository, rule, branch, or triage action.
+* **Triage the finding.** If the finding is not useful or important, it can be deprioritized through triaging. Triage actions include commenting and ignoring. This is one method to reduce **false positives**.
```

Comment:
```suggestion
* **Triage the finding**. If the finding is not useful or important, it can be deprioritized through triaging. Triage actions include commenting and ignoring. This is one method to reduce **false positives**.
```